### PR TITLE
Marketplace Rebranding

### PR DESCRIPTION
**Pull request by jhadobe**
This pull request (PR) updates the Marketplace branding and links to accommodate upcoming rebranding.
Staging build: https://developer-stage.adobe.com/commerce/marketplace/

### CODE REVIEW

File: src/pages/guides/eqp/v1/access-keys.md [hunk 1]
```diff
@@ -20,8 +20,8 @@ Marketplace EQP API **access keys** are self-serviced.  As long as you are [elig
 ## How do I create an API access key
 
 -  You create your API access key from one, or both, of the **Marketplace Developer Portal** user interfaces:
-   -  production - [https://developer.magento.com][1]
-   -  sandbox - [https://developer-stg.magento.com][2]
+   -  production - [https://commercedeveloper.adobe.com][1]
+   -  sandbox - [https://commercedeveloper-sandbox.adobe.com][2]
 
 ### Create a new API access key
```
Comment (AndreaFernandezArmendariz) [inline src/pages/guides/eqp/v1/access-keys.md:26]: update Images with the latest UI
For example for step 3, the current experience is the following: 

image

Please, update also the images for steps 4, 6 and 7.

File: src/pages/guides/eqp/v1/help.md [hunk 2]
```diff
@@ -23,14 +23,14 @@ Use these resources if you need additional help.
 
 **Websites:**
 
--  [Marketplace Developer Portal](https://developer.magento.com)
--  [Sandbox Marketplace Developer Portal](https://developer-stg.magento.com)
--  [Commerce Marketplace](https://marketplace.magento.com)
+-  [Marketplace Developer Portal](https://commercedeveloper.adobe.com)
+-  [Sandbox Marketplace Developer Portal](https://commercedeveloper-sandbox.adobe.com)
+-  [Commerce Marketplace](https://commercemarketplace.adobe.com)
 -  [Magento Account](https://account.magento.com)
```
Comment (AndreaFernandezArmendariz) [inline src/pages/guides/eqp/v1/help.md:29]: Could we please change "Magento account" to "Commerce Marketplace account"?

File: src/pages/guides/eqp/v1/index.md [hunk 3]
```diff
@@ -18,8 +18,8 @@ The **Marketplace Developer Portal** provides you the capability to do the follo
 
 There are two separate **Marketplace Developer Portal** environments where the Marketplace EQP API can be used:
 
--  Production - [https://developer.magento.com][1]
--  Sandbox - [https://developer-stg.magento.com][2]
+-  Production - [https://commercedeveloper.adobe.com][1]
+-  Sandbox - [https://commercedeveloper-sandbox.adobe.com][2]
 
 Use your same login information for the production **Marketplace Developer Portal** to also log into the **Sandbox Marketplace Developer Portal**. Learn more about the [sandbox](sandbox.md) environment.
```
Comment (AndreaFernandezArmendariz) [inline src/pages/guides/eqp/v1/index.md:25]: Section "API availability" - we need to update the list of partners who are eligible. Please remove the table and add the following list: 

* Adobe Platinum Technology Partner 
* Adobe Gold Technology Partner
* Adobe Silver Technology Partner
* Adobe Platinum Solution Partner 
* Adobe Gold Solution Partner
* Adobe Silver Solution Partner 
* Adobe Bronze Solution Partner

File: src/pages/guides/eqp/v1/sandbox.md [hunk 4]
```diff
@@ -36,5 +36,5 @@ The following list highlights some of the key differences between the sandbox an
 -  [Session tokens](auth.md#how-to-use-a-session-token) that are generated in the **sandbox** are separate and distinct from **production**. You cannot use session tokens from one environment in the other environment.
```
Comment (AndreaFernandezArmendariz) [inline src/pages/guides/eqp/v1/sandbox.md:36]: Update image on this page with latest UI. We can use the following: 

image

**Review by AndreaFernandezArmendariz — COMMENTED** [review]
Thanks @jhadobe for all these changes. 
There are some additional updates that the team needs to implement for the UI Refresh that will impact some images that we have in our documentation. Once they are implemented we can update those. Great work!

File: src/pages/guides/eqp/v1/help.md [hunk 5]
```diff
@@ -9,7 +9,7 @@ Use these resources if you need additional help.
 
 **Slack:**
 
--  workspace: [Magento Community Engineering](https://opensource.magento.com/slack)
+-  workspace: [Magento Community Engineering](https://developer.adobe.com/open/magento/slack)
```
Comment (AndreaFernandezArmendariz) [inline src/pages/guides/eqp/v1/help.md:12]: The name of the workspace is **Magento Open Source**  magentocommeng.slack.com 

image

**Review by AndreaFernandezArmendariz — APPROVED** [review]
+1 thanks for adding all these changes. 
We will wait for the team to do some implementations until we close this task. :-)
The goal is to release all these changes on June 13th.

**Review by keharper — APPROVED** [review]

**Pull request merged**